.github/workflows/get_release_notes.yml

```diff
@@ -0,0 +1,52 @@
+name: Get Release notes
+on: [workflow_dispatch]
+jobs:
+  release_notes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get the list of allowed pull request labels
+        run: |
+          RED='\033[0;31m'
+          NC='\033[0m'
+          FILE_NAME=.pull-requests-allowed-labels-list
+          FILE_PATH=$PROJECT_ROOT/$FILE_NAME
+          if [[ ! -f "$FILE_PATH" ]]; then
+             echo -e "${RED}$FILE_NAME file doesn't exits in the root of the respository${NC}"
+             exit 1
+          fi 
+          echo "LABELS=$(cat $FILE_PATH)" >> $GITHUB_OUTPUT
+        id: get-allowed_labels
+        env:
+          PROJECT_ROOT: ${{ github.workspace }}
+      - name: Generate release notes
+        run: |
+          LATEST_TAG=$(git describe --tags --abbrev=0)
+          NUMBER_OF_COMMITS_SINCE_LAST_RELEASE=$(git log --oneline $LATEST_TAG..HEAD | wc -l | sed 's/^ *//g' | sed 's/ *$//g')
+
+          IFS=',' read -r -a ALLOWED_LABELS_ARRAY <<< "$ALLOWED_LABELS"
+
+          OUTPUT=""
+
+          for LABEL in "${ALLOWED_LABELS_ARRAY[@]}"; do
+             LABEL=$(echo "$LABEL" | sed 's/^ *//g' | sed 's/ *$//g')
+             GIT_LOG_CMD="git log -$NUMBER_OF_COMMITS_SINCE_LAST_RELEASE"
```

Comment:
This is `git log`, how does it tae description from GitHub PRs? 🤔

Reply:
the PR body is added to the merge commit message when merging the PR automatically, we just need to remember not to modify when merging